modules/core/keeper/msg_server.go

```diff
@@ -291,9 +301,11 @@ func (k Keeper) ChannelOpenTry(goCtx context.Context, msg *channeltypes.MsgChann
 		return nil, sdkerrors.Wrap(err, "could not retrieve module from port-id")
 	}
 
-	_, channelID, cap, err := channel.HandleMsgChannelOpenTry(ctx, k.ChannelKeeper, portCap, msg)
+	channelID, cap, err := k.ChannelKeeper.ChanOpenTry(ctx, msg.Channel.Ordering, msg.Channel.ConnectionHops, msg.PortId, msg.PreviousChannelId,
+		portCap, msg.Channel.Counterparty, msg.Channel.Version, msg.CounterpartyVersion, msg.ProofInit, msg.ProofHeight,
+	)
 	if err != nil {
-		return nil, err
+		return &channeltypes.MsgChannelOpenTryResponse{}, sdkerrors.Wrap(err, "channel handshake open try failed")
```

Comment:
Return nil here, since that's what the rest of the file does

Reply:
Updated :+1: